### DISCUSSION TITLE
(Node native global) fetch is not defined  no-undef

### DISCUSSION THREAD

**Discussion by fdiasr** [eslint/eslint#15841] [Q&A] [answered]
I am using in anew project Node V18 to use new fetch implementation. Running ESLint I got this message error ('fetch' is not defined  no-undef').

I fixed configuring .eslintrc with globals for fetch, but I'd like to know if it can defined into eslint rules and available in a next version to avoid a manual configuration. If there is a documentation or something explaining how to do that, I d like to see and also I can open a PR for that

**Comment by aladdin-add** [comment, marked as answer]
eslint's globals were from the package [globals](https://github.com/sindresorhus/globals). There is an issue about adding the new global `fetch`: https://github.com/sindresorhus/globals/issues/188. please feel free to send a PR. :)

**Reply by fdiasr** [reply]
https://github.com/sindresorhus/globals/pull/189
